### PR TITLE
chore: use node lts 

### PR DESCRIPTION
**Pull request by kdamball**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-v16.13.1
+v16.14.0
```
Comment (shikaan) [inline .nvmrc:1]: Shall we make this v16 so it always picks latest LTS?

Comment (shikaan) [reply .nvmrc:1]: (same above)

**Comment by andipaetzold** [comment]
Why not just merge master?

File: .nvmrc (hunk 1 above)
Comment (kdamball) [reply .nvmrc]: Even better - it allows `lts` as an alias :)

**Review by shikaan — APPROVED** [review]

**Comment by kdamball** [comment]
@andipaetzold we might also need this change for canary, right?

**Comment by andipaetzold** [comment]
Yes, but why not merge master into canary? I did the same here https://github.com/contentful/create-contentful-app/pull/882

**Pull request merged**